recipe/meta.yaml

```diff
@@ -29,7 +27,7 @@ requirements:
     - setuptools !=74.0.0,!=74.1.0,!=74.1.1,!=74.1.2
     - wheel
     - maturin >=1.8.6,<2
-    - cffi >=1.14
+    - cffi 1.17.1
```

Comment:
We can drop a such strict pinning for the Python package `cffi`. It can be difficult to make a mass build-out for new Python versions.
We can still use `cffi >=1.14`